src/EnergyPlus/SimulationManager.cc

```diff
@@ -1042,73 +1040,108 @@ namespace SimulationManager {
+                            // Which happens if you put an "empty" entry in the extensible portion
+                            auto it = diagnosticsExtensible.find("key");
+                            if (it == diagnosticsExtensible.end()) {
+                                ShowWarningError(CurrentModuleObject + ": empty key found, consider removing it to avoid this warning.");
+                                continue;
+                            }
+                            std::string diagnosticName = *it;
+
+                            if (UtilityRoutines::SameString(diagnosticName, "DisplayExtraWarnings")) {
+                                DisplayExtraWarnings = true;
+                            } else if (UtilityRoutines::SameString(diagnosticName, "DisplayAdvancedReportVariables")) {
+                                DisplayAdvancedReportVariables = true;
+                            } else if (UtilityRoutines::SameString(diagnosticName, "DisplayAllWarnings")) {
+                                DisplayAllWarnings = true;
+                                DisplayExtraWarnings = true;
+                                DisplayUnusedObjects = true;
+                                DisplayUnusedSchedules = true;
+                            } else if (UtilityRoutines::SameString(diagnosticName, "DisplayUnusedObjects")) {
+                                DisplayUnusedObjects = true;
+                            } else if (UtilityRoutines::SameString(diagnosticName, "DisplayUnusedSchedules")) {
+                                DisplayUnusedSchedules = true;
+                            } else if (UtilityRoutines::SameString(diagnosticName, "DisplayZoneAirHeatBalanceOffBalance")) {
+                                DisplayZoneAirHeatBalanceOffBalance = true;
+                            } else if (UtilityRoutines::SameString(diagnosticName, "DoNotMirrorDetachedShading")) {
+                                MakeMirroredDetachedShading = false;
+                            } else if (UtilityRoutines::SameString(diagnosticName, "DoNotMirrorAttachedShading")) {
+                                MakeMirroredAttachedShading = false;
+                            } else if (UtilityRoutines::SameString(diagnosticName, "IgnoreInteriorWindowTransmission")) { // TODO: Not a valid key choice
+                                IgnoreInteriorWindowTransmission = true;
+                            } else if (UtilityRoutines::SameString(diagnosticName, "ReportDuringWarmup")) {
+                                ReportDuringWarmup = true;
+                            } else if (UtilityRoutines::SameString(diagnosticName, "DisplayWeatherMissingDataWarnings")) {
+                                DisplayWeatherMissingDataWarnings = true;
+                            } else if (UtilityRoutines::SameString(diagnosticName, "IgnoreSolarRadiation")) { // TODO: Not a valid key choice
```

Comment:
Eventually, it would be good to enable these in the IDD and document them, or determine if any are useless and remove them entirely. 
@xuanluo113 says that some of these are not used in the code. From https://github.com/NREL/EnergyPlus/issues/7960#issuecomment-622206109:

> TimingFlag, CreateMinimalSurfaceVariables and CreateNormalSurfaceVariables are not used anymore in the code. Maybe they could be removed from the input check as well as the system variable list.